### ISSUE TITLE
self_accessor caused broken replacement

### ISSUE THREAD

**Issue by glensc** [FriendsOfPHP/PHP-CS-Fixer#4307]
- With PHP CS Fixer 2.14.1 Sunrise by Fabien Potencier and Dariusz Ruminski (2e82abd)

```
   1) /Users/glen/scm/eventum/eventum/lib/eventum/class.project.php (self_accessor)
```

```diff
diff --git a/lib/eventum/class.project.php b/lib/eventum/class.project.php
index bfd2ac0ff..6fe26fec4 100644
--- a/lib/eventum/class.project.php
+++ b/lib/eventum/class.project.php
@@ -1088,6 +1088,6 @@ class Project


         return $repo
             ?: $repo = Doctrine::getEntityManager()
-                ->getRepository(Entity\Project::class);
+                ->getRepository(Entityself::class);
     }
 }
```

basically class in root namespace:

```php

use Eventum\Model\Entity;

class Project {
}

```

referenced `Entity\Project::class` which got replaced with `Entityself::class`.

**Comment by glensc** [comment]
i'm sorry, in what version this is supposed to be fixed? still broken with 2.14.2

**Comment by SpacePossum** [comment]
The last phar releases are from 17 Feb., this fix is from after that I think. So it is fixed in all supported branches, but no releases have been made from these for a while.
